### PR TITLE
sheCode 2018 Update

### PR DESCRIPTION
**Pull request by Sech1**
There may still be some small changes to the site, (FAQ, times, etc) but this has the working form and most other updates finished.

### CODE REVIEW

File: config/mail.php [hunk 1]
```diff
@@ -56,8 +56,8 @@
     */
 
     'from' => [
-        'address' => env('MAIL_FROM_ADDRESS', 'hello@example.com'),
-        'name' => env('MAIL_FROM_NAME', 'Example'),
+        'address' => env('MAIL_FROM_ADDRESS', 'no-reply@shecode.cs.siue.edu'),
```
Comment (ryanvade) [inline config/mail.php]: I can't accept these changes in config/mail.php.  Referencing the Name and address for the mail settings should not be included with the repository, rather they need to be included in the installed configuration provided by the .env file. The .env file is already configured to provide these settings. See [Configuration - Laravel Docs](https://laravel.com/docs/5.5/configuration#environment-configuration)

File: resources/views/join.blade.php [hunk 2]
```diff
@@ -45,6 +49,14 @@
       </span>
         </p>
       </div>
+
+      <div class="field">
+        <label class="label">Allergies / Food Preferences</label>
+        <div class="control">
+          <textarea id="allergies" class="input" rows="5" placeholder="List All Allergies" name="allergies" onchange="hideErrors()"></textarea>
```
Comment (ryanvade) [inline resources/views/join.blade.php]: PLease use this change for the textarea (info [HERE](https://bulma.io/documentation/form/textarea/))
```suggestion
          <textarea id="allergies" class="textarea" rows="5" placeholder="List All Allergies" name="allergies" onchange="hideErrors()"></textarea>
```

File: config/mail.php (hunk 1 above)
Comment (ryanvade) [reply config/mail.php]: I also cannot accept the PDFs. I understand the reason for including them but if this repo is used for another few years then more and more PDFs will be included.  This leads to repo junk. Please remove.

**Review by ryanvade — CHANGES_REQUESTED** [review]
Overall, I cannot accept. Please see the list of changes.

File: resources/views/index.blade.php [hunk 3]
```diff
@@ -10,7 +10,7 @@
     <div class="container has-text-centered">
       <img src="/images/shecodeWhite.png" alt="SheCode" width="50%">
       <h2 class="subtitle">
-        Saturday Decemeber 2nd, 2017<br>
+        Saturday Decemeber 1st, 2018<br>
```
Comment (ryanvade) [inline resources/views/index.blade.php]:
```suggestion
        Saturday December 1st, 2018<br>
```

**Review by ryanvade — CHANGES_REQUESTED** [review]

**Comment by Sech1** [comment]
Sorry I haven’t had time to make these changes yet this week, I have two tests (stat and programming languages) and work.
I will get to it when I can.

Jared Schooley

On Oct 22, 2018, at 8:11 PM, Ryan Owens <notifications@github.com<mailto:notifications@github.com>> wrote:


@ryanvade requested changes on this pull request.

________________________________

In resources/views/index.blade.php<https://github.com/ryanvade/shecode/pull/1#discussion_r227189067>:

> @@ -10,7 +10,7 @@
     <div class="container has-text-centered">
       SheCode
       <h2 class="subtitle">
-        Saturday Decemeber 2nd, 2017<br>
+        Saturday Decemeber 1st, 2018<br>


⬇️ Suggested change

-        Saturday Decemeber 1st, 2018<br>
+        Saturday December 1st, 2018<br>

**Comment by Sech1** [comment]
Your requested changes should be added/complete

**Pull request merged**